### PR TITLE
UIDEXP-137: Refactor to MirageJS

### PR DESCRIPTION
**Pull request by minkimcello**
UIDEXP-137

## Motivation
Screen Shot 2020-07-14 at 4 57 15 PM
This is part of an on-going initiative to completely migrate all of `folio-org` projects from `@bigtest/mirage` to `miragejs`.

[`@bigtest/mirage`](https://github.com/bigtestjs/mirage/) was originally a prototype to make `mirage` available to any framework, but it has since been archived in favor of [`miragejs`](https://miragejs.com/). The documentation for `miragejs` is not only prodigious for its extensive API docs but also for its tutorials and guides. Furthermore, it receives regular bug-fixes and maintenance releases in thanks to its extremely large and active community.

Previous PR: [@folio-org/ui-inventory #1110](https://github.com/folio-org/ui-inventory/pull/1110)
_Linking the previous pull request so that subsequent ones can be found tagged below._

<details>
<summary>Overview of progress at the time of the creation of this PR (in chronological order):</summary>
<br>
✅ <code>ui-checkin</code>
<br>
☑️ <code>ui-checkout</code>
<br>
☑️ <code>ui-circulation</code>
<br>
☑️ <code>ui-calendar</code>
<br>
☑️ <code>ui-agreements</code>
<br>
☑️ <code>ui-app-template</code>
<br>
☑️ <code>ui-inventory</code>
<br>
☑️ <code>ui-data-export</code>
</details>

## Approach
- [x] Cloned, installed, and then ran tests to double check that the tests are passing before refactoring.
- [x] Removed `@bigtest/mirage` and added `inflected`, `faker`, and `miragejs`.
- [x] Ran tests after the refactor to confirm they all pass.
- [x] ESLinted

### CODE REVIEW

**Comment by VictorSoroka1** [comment]
Please handle eslint warnings.

**Comment by VictorSoroka1** [comment]
- Reference the ticket number both in the title of the PR and the description UIDEXP-137. See the https://github.com/folio-org/ui-data-export/pull/109 for reference.
- Update the changelog - https://github.com/folio-org/ui-data-export/blob/master/CHANGELOG.md

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # Change history for ui-data-export
 
 ## 2.1.0 (IN PROGRESS)
+* Refactor to miragejs from bigtest/mirage. UIDEXP-137.
```
Comment (VictorSoroka1) [inline CHANGELOG.md:4]: This one should be at the bottom right after `Add job profiles deleting. UIDEXP-111.`

**Comment by minkimcello** [comment]
@VictorSoroka1 I'm unable to merge because I don't have write access. Could you do it for me?

**Pull request merged**